### PR TITLE
fix: grammatical errors

### PR DESCRIPTION
**Pull request by nitinknolder**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -179,13 +178,13 @@ Samples: (even more [samples](https://github.com/angular/angular/commits/master)
...
 ```
-fix(release): need to depend on latest rxjs and zone.js
+fix(release): need to depend on the latest rxjs and zone.js
 
 The version in our package.json gets copied to the one we publish, and users need the latest of these.
 ```
 
 ### Revert
-If the commit reverts a previous commit, it should begin with `revert: `, followed by the header of the reverted commit. In the body it should say: `This reverts commit <hash>.`, where the hash is the SHA of the commit being reverted.
+If the commit reverts a previous commit, it should begin with `revert: `, followed by the header of the reverted commit. In the body, it should say: `These reverts commit <hash>.`, where the hash is the SHA of the commit being reverted.
```
Comment (petebacondarwin) [inline CONTRIBUTING.md:187]: this one is not correct

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -253,7 +252,7 @@ The body should include the motivation for the change and contrast this with pre
 The footer should contain any information about **Breaking Changes** and is also the place to
 reference GitHub issues that this commit **Closes**.
 
-**Breaking Changes** should start with the word `BREAKING CHANGE:` with a space or two newlines. The rest of the commit message is then used for this.
+**Breaking Changes** should start with the word `BREAKING CHANGE:` with space or two newlines. The rest of the commit message is then used for this.
```
Comment (petebacondarwin) [inline CONTRIBUTING.md:255]: this one is not correct

File: docs/DEVELOPER.md [hunk 3]
```diff
@@ -136,14 +136,14 @@ $ yarn gulp lint
 
 When a build of any branch on the upstream fork angular/angular is green on CircleCI, it
 automatically publishes build artifacts to repositories in the Angular org, eg. the `@angular/core`
-package is published to https://github.com/angular/core-builds.
+the package is published to https://github.com/angular/core-builds.
```
Comment (petebacondarwin) [inline docs/DEVELOPER.md:139]: the "the" is on the previous line

File: docs/DEVELOPER.md [hunk 4]
```diff
@@ -136,14 +136,14 @@ $ yarn gulp lint
...
 
 You may find that your un-merged change needs some validation from external participants.
 Rather than requiring them to pull your Pull Request and build Angular locally, they can depend on
 snapshots of the Angular packages created based on the code in the Pull Request.
 
 ### Getting Packages from Build Artifacts
-Each CI run for a Pull Request stores the built Angular packages as
+Each CI run for Pull Request stores the built Angular packages as
```
Comment (petebacondarwin) [inline docs/DEVELOPER.md:146]: This one is not correct

File: docs/DEVELOPER.md [hunk 5]
```diff
@@ -225,7 +225,7 @@ It will automatically recognize `*.bazel` and `*.bzl` files.
 
 ### Remote Build Execution and Remote Caching
 Bazel builds in the Angular repository use a shared cache.  When a build occurs a hash of the inputs is computed
-and checked against available outputs in the shared cache.  If an output is found, it is used as the output for the
+and checked against available outputs in the shared cache.  If the output is found, it is used as the output for the
```
Comment (petebacondarwin) [inline docs/DEVELOPER.md:228]: I'm not sure this is right

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -55,7 +55,7 @@ We want to fix all the issues as soon as possible, but before fixing a bug we ne
 
 A minimal reproduction allows us to quickly confirm a bug (or point out a coding problem) as well as confirm that we are fixing the right problem.
 
-We will be insisting on a minimal reproduction scenario in order to save maintainers time and ultimately be able to fix more bugs. Interestingly, from our experience, users often find coding problems themselves while preparing a minimal reproduction. We understand that sometimes it might be hard to extract essential bits of code from a larger codebase but we really need to isolate the problem before we can fix it.
+We will be insisting on a minimal reproduction scenario in order to save maintainer's time and ultimately be able to fix more bugs. Interestingly, from our experience, users often find coding problems themselves while preparing a minimal reproduction. We understand that sometimes it might be hard to extract essential bits of code from a larger codebase but we really need to isolate the problem before we can fix it.
```
Comment (petebacondarwin) [inline CONTRIBUTING.md:58]: I think this should actually be `maintainers'`

File: docs/DEVELOPER.md (hunk 5 above)
Comment (nitinknolder) [reply docs/DEVELOPER.md:228]: `an` may be redundant when used with uncountable noun output in this sentence so i consider removing it.

Comment (petebacondarwin) [reply docs/DEVELOPER.md:228]: It is not uncountable in this context. See the previous sentence where it says `available outputs`.

**Comment by petebacondarwin** [comment]
Oh - and thank you providing this PR, by the way. I hope my comments/suggestions did not come across too harshly!

**Comment by nitinknolder** [comment]
> Oh - and thank you providing this PR, by the way. I hope my comments/suggestions did not come across too harshly!

Your Welcome, no not at all!

**Review by petebacondarwin — APPROVED** [review]
Reviewed-for: global-docs-approvers

**Comment by petebacondarwin** [comment]
Oh and can you please rebase on master - see https://app.circleci.com/pipelines/github/angular/angular/17225/workflows/dcd17fa3-fcae-4992-a28f-6548e2c7ba8b/jobs/741791/steps

**Comment by petebacondarwin** [comment]
And... (sorry) the commit messages must follow our syntax, as described (ironically) in our https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit. I would suggest just squashing all your commits into a single one during your rebase. The commit message body needs to be more than 100 characters long. Something like:

```
docs: fix grammatical errors in developer docs

The CONTRIBUTOR and DEVELOPER markdown docs contained a few typos
and grammatical errors, which are fixed by this commit.
```

**Review by petebacondarwin — APPROVED** [review]

**Comment by nitinknolder** [comment]
> Oh and can you please rebase on master - see https://app.circleci.com/pipelines/github/angular/angular/17225/workflows/dcd17fa3-fcae-4992-a28f-6548e2c7ba8b/jobs/741791/steps

I have executed all the commands mention in the steps given above but it still showing build failure please help.

**Comment by petebacondarwin** [comment]
You need to rebase on the upstream master (i.e. the angular/angular master, not your local master nor your own fork). Something like

```bash
git checkout master
git pull upstream master
git checkout fix-grammatical-errors
git rebase master
```

**Comment by nitinknolder** [comment]
> You need to rebase on the upstream master (i.e. the angular/angular master, not your local master nor your own fork). Something like
> 
> ```shell
> git checkout master
> git pull upstream master
> git checkout fix-grammatical-errors
> git rebase master
> ```

Thanks it worked but some test cases are failing and i think it's not because of my PR please have a look and let me know.

**Comment by petebacondarwin** [comment]
Rerunning the failed test. It is just a bit flaky

**Comment by nitinknolder** [comment]
> Rerunning the failed test. It is just a bit flaky

Oh! thanks np.

**Comment by nitinknolder** [comment]
> Rerunning the failed test. It is just a bit flaky

I think you need to rerun the build it's still not working.

**Review by gkalpak — APPROVED** [review]

**Review by devversion — APPROVED** [review]

**Pull request closed without merging**